Runtime/Exporters/JsonExporter.cs

```diff
@@ -25,7 +25,7 @@ private void Reset()
 
         public override void Export()
         {
-            string jsonPath = Path.Combine(Map.exportSettings?.OutputDirectory, "data.json").ToOSSpecificPath();
+            string jsonPath = Path.Combine(Map.projectFolder, ".treasured", Map.projectFolder, "data.json").ToOSSpecificPath();
```

Comment:
Should be using Map.exportSettings.OutputDirectory to make it backward compatible